near-sdk/src/store/unordered_map/mod.rs

```diff
@@ -630,6 +630,23 @@ where
     }
 }
 
+impl<K, V, H> UnorderedMap<K, V, H>
+where
+    K: BorshSerialize + BorshDeserialize + Ord + Clone,
+    V: BorshSerialize + BorshDeserialize,
+    H: ToKey,
+{
+    pub fn defrag(&mut self) {
+        self.keys.defrag(|key, new_index| {
+            // Check if value is in map to replace first
+            let entry = self.values.get_mut_inner(key);
+            if let Some(existing) = entry.value_mut() {
```

Comment:
This works just fine unless I miss something:

```suggestion
            if let Some(existing) = self.values.get_mut(key) {
```

Reply:
Yeah, looks like it does the exact same thing under the hood :+1: